candi.sh

```diff
@@ -605,6 +599,40 @@ guess_platform() {
     fi
 }
 
+guess_ostype() {
+    # Try to guess the operating system type (ostype)
+    if [ -f /usr/bin/cygwin1.dll ]; then
+        echo cygwin
+
+    elif [ -f /etc/fedora-release ]; then
+	    echo linux
+    
+    elif [ -f /etc/redhat-release ]; then
+        echo linux
+    
+    elif [ -x /usr/bin/sw_vers ]; then
+        echo macos
+    
+    elif [ -x /usr/bin/lsb_release ]; then
+        local DISTRO=$(lsb_release -i -s)
+        local CODENAME=$(lsb_release -c -s)
+        local DESCRIPTION=$(lsb_release -d -s)
+        case ${DISTRO}:${CODENAME}:${DESCRIPTION} in
+            *:*:*Ubuntu*\ 12*)     echo linux;;
```

Comment:
It is the last option for candi, centos/rhel are found earlier.
I simply copied the guess_platform function and simplified it in this place here.